app/src/main/java/com/example/ar_reshare/FeedActivity.java

```diff
@@ -45,11 +48,16 @@ public class FeedActivity extends AppCompatActivity {
     // Global Recycler View
     RecyclerView recyclerView;
 
+    // CountDownLatch to ensure thread only works after results have been received from backend
+    private CountDownLatch readyLatch;
+
+   // FusedLocationProviderClient fusedLocationClient =
```

Comment:
Can you please remove these redundant comments?

Reply:
That's code that is in progress still! Just wanna push this out so that everyone can access getProfileById 😄